hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: minor
+
+Using an unordered collection with the ``permutations`` strategy has been
```

Comment:
:func:`~hypothesis.strategies.permutations`

Use the above syntax to generate a nicely-formatted link to the docs for `permutations`.

I'd also like a brief explanation of *why* we made this change, e.g.:

> ...deprecated, because the order that e.g. a set shrinks to is arbitrary and shrinking may therefore give different results between runs.